### ISSUE TITLE
Support built-in trash feature of Android 11 and above

### ISSUE THREAD

**Issue by t-m-w** [SimpleMobileTools/Simple-Gallery#2343]
Simple Gallery has a built-in Recycle Bin feature, but Android has native support for trashing items since Android 11 through the Media Store API. In practice, this tends to work by the file being renamed to .trashed-[datetime code]-[original file name], setting the IS_TRASHED flag to true in the Media Store, and keeping the file in its original folder.

Please consider supporting the native trash feature.

Here are some references:
- [XDA-Developers: Android 11 has a hidden Recycle Bin for trashed photos and videos](https://www.xda-developers.com/android-11-hidden-recycle-bin-trashed-photos-videos)
- [Android Developer Documentation: MediaStore.QUERY_ARG_MATCH_TRASHED](https://developer.android.com/reference/android/provider/MediaStore#QUERY_ARG_MATCH_TRASHED)
- [Android Developer Documentation: MediaStore.createTrashRequest](https://developer.android.com/reference/android/provider/MediaStore#createTrashRequest(android.content.ContentResolver,%20java.util.Collection%3Candroid.net.Uri%3E,%20boolean))
- [Android Developer Documentation: MediaColumns.IS_TRASHED](https://developer.android.com/reference/android/provider/MediaStore.MediaColumns#IS_TRASHED)
- [YouTube: Storage access with Android 11](https://www.youtube.com/watch?v=RjyYCUW-9tY&feature=youtu.be)

**Comment by tibbi** [comment]
was checking it already, but you have to confirm every file that gets deleted

**Comment by dtbj** [comment]
I agree with you @tibbi It's UX is so bad that accept deny popup is ultra annoying.

**Comment by t-m-w** [comment]
Does this opinion rule out support for even *viewing* the native trash, or should viewing the trash and actually utilizing the trash for deletions be considered separate requests?
